src/javaRestTest/java/com/o19s/es/ltr/logging/LoggingIT.java

```diff
@@ -623,7 +623,7 @@ protected void assertSearchHitsExtraLogging(Map<String, Doc> docs, SearchRespons
             Doc d = docs.get(hit.getId());
 
             assertEquals(6, log1.size());
-            assertEquals(6, log2.size());
```

Comment:
This test fails for me with the change in this PR.
Any specific reason that this change was necessary for you?